src/pymorize/cli.py

```diff
@@ -232,6 +238,22 @@ def directory(config_file, output_dir, verbose, quiet, logfile, profile_mem):
         cmorizer.check_rules_for_output_dir(output_dir)
 
 
+################################################################################
+################################################################################
+################################################################################
+
+################################################################################
+# COMMANDS FOR scripts
+################################################################################
+
+
+@scripts.group()
+def fesom1():
+    pass
+
+
+fesom1.add_command(convert, name="nodes-to-levels")
+
```

Comment:
It seems that this is not used, can it be cleaned up, or finalized?

Reply:
as above

Reply:
I looked at these lines again (https://github.com/esm-tools/pymorize/pull/77/files#diff-f7f0ca36502b11f2609ec32f5c6b4d304ab0f7e7e9438a689af344588ef7eaeaR208-R210), now I get it how it works. Thanks for the clarification!